domaci.html

```diff
@@ -1,46 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Document</title>
 
     <style>
-    p {        font-size: 15px 
-    }
-   button{background-color:green; 
-} 
-    .button {color:white}
-    a {text-decoration: none} 
-    
+      p {
+        font-size: 15px;
+      }
+      button {
+        background-color: green;
+      }
+      .button {
+        color: white;
+      }
+      a {
+        text-decoration: none;
+      }
     </style>
-</head>
-<body>
-        <form action="../../form-result.php" method="post" target="_blank">
+  </head>
+  <body>
+    <form action="../../form-result.php" method="post" target="_blank">
+      <p><label for="username">Username: </label></p>
```

Comment:
No need to wrap every tag in a `p`. Google the `br` tag or make these elements `display: block`